src/kirby/components/calendar/calendar.webview.html

```diff
@@ -0,0 +1,200 @@
+<!doctype html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <title>Calendar</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">
+        <link rel="stylesheet" type="text/css" href="calendar.webview.css">
+        <script src="nativescript-webview-interface.js"></script>
+        <script>
+            (function () {
```

Comment:
Since we've already considered limited ourselves to EsNext (greenfield browsers), I'd suggest that we make use of lexical scoping (`let`, `const`) as well as lambdas. Currently the code resembles ES5

Reply:
Hmmmm - that might be, I don't have time to change that now though.... 😢